ts_scripts/sanity_utils.py

```diff
@@ -184,6 +184,13 @@ def test_sanity():
         print(f"## Failed to run inference on {resnet18_model['name']} model")
         sys.exit(1)
 
+    response = ts.unregister_model(resnet18_model["name"])
```

Comment:
are changes to this file related to this PR?

Reply:
I observed that if I run sanity tests back to back, the registration for `resnet-18` version `1` fails because the model already exits. In the sanity test logic, we clean up all other models, so I added this segment to unregister the `resnet-18` model as well after testing load from snapshot.

Reply:
Would it be better to include this change in a separate PR? CI should still work without this change too.

Reply:
Na I think making small improvements as you see them is a good idea, please keep the change